ludwig/hyperopt/execution.py

```diff
@@ -32,7 +32,7 @@
 from ludwig.callbacks import Callback
 from ludwig.constants import MAXIMIZE, TEST, TRAINER, TRAINING, TYPE, VALIDATION
 from ludwig.hyperopt.results import HyperoptResults, TrialResults
-from ludwig.hyperopt.search_algos import get_search_algorithm
+from ludwig.hyperopt.search_algos import instantiate_search_algorithm  # implicit import
```

Comment:
Should this be

`from ludwig.hyperopt.registry import instantiate_search_algorithm`?

Reply:
yup, I thought that this implicit import would work but this function is actually not imported in `search_algos` so it fails.